openmls/src/messages/mod.rs

```diff
@@ -305,6 +317,71 @@ impl SignedStruct<GroupInfoTBS> for GroupInfo {
     }
 }
 
+/// A type that represents a group info of which the signature has not been verified.
+/// It implements the [Verifiable] trait and can be turned into a group info by calling
```

Comment:
I'm thinking if it would not be easier to open an issue for things like that and do a single PR that rectifies all of this close to some release. I will rectify it here, though. Just a thought.